google-cloud/google-cloud.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.0.0"
 
   gem.add_dependency "google-cloud-bigquery", "~> 0.30.0"
+  gem.add_dependency "google-cloud-dataproc", "~> 0.1.0"
```

Comment:
This dependency should not be added until the gem is released.

Reply:
Removed.